src/interfaces/email/email.interface.ts

```diff
@@ -0,0 +1,36 @@
+import { EmailPlatformConstants } from '@constants/email';
+import { Transporter, SentMessageInfo } from 'nodemailer';
+
+export interface ISendGridOptions {
+  auth: AuthSendGrid;
+}
+
+export interface IEmail {
+  from: string;
+  to: string;
+  subject: string;
+  template?: string;
+  text?: string;
+  context?: any;
+}
+
+export interface AuthSendGrid {
```

Comment:
probably`IAuthSendGrid`. I also used the `I` convention to name interfaces, but recently I was reading clean code and they a different point of view, if you want to check it out :

http://www.amygdalum.net/en/naming-interface-implementation.html

https://softwareengineering.stackexchange.com/questions/117348/should-interface-names-begin-with-an-i-prefix

Reply:
oh! very useful information! I'll take a look at it, for now I changed it to `IAuthSendGrid`.
thanks for your review!